CMakeLists.txt

```
@@ -535,14 +561,48 @@ endif()
 #        Installation section         #
 #######################################
 
-install(DIRECTORY config DESTINATION ${DATA_DIR})
-install(DIRECTORY Mods DESTINATION ${DATA_DIR})
+if(ANDROID)
+	if(ANDROID_STL MATCHES "_shared$")
+		set(stlLibName "${CMAKE_SHARED_LIBRARY_PREFIX}${ANDROID_STL}${CMAKE_SHARED_LIBRARY_SUFFIX}")
+		install(FILES "${CMAKE_SYSROOT}/usr/lib/${ANDROID_SYSROOT_LIB_SUBDIR}/${stlLibName}"
+			DESTINATION ${LIB_DIR}
+		)
+	endif()
+
+	install(FILES AUTHORS
+		DESTINATION res/raw
+		RENAME authors.txt
+	)
+
+	# zip internal assets - 'config' and 'Mods' dirs, save md5 of the zip
+	install(CODE "
+		cmake_path(ABSOLUTE_PATH CMAKE_INSTALL_PREFIX
+			OUTPUT_VARIABLE absolute_install_prefix
+		)
+		set(absolute_data_dir \"\${absolute_install_prefix}/${DATA_DIR}\")
+		file(MAKE_DIRECTORY \"\${absolute_data_dir}\")
+
+		set(internal_data_zip \"\${absolute_data_dir}/internalData.zip\")
+		execute_process(COMMAND
+			\"${CMAKE_COMMAND}\" -E tar c \"\${internal_data_zip}\" --format=zip -- config Mods
+			WORKING_DIRECTORY \"${CMAKE_SOURCE_DIR}\"
+		)
+
+		file(MD5 \"\${internal_data_zip}\" internal_data_zip_md5)
+		file(WRITE \"\${absolute_data_dir}/internalDataHash.txt\"
+			\${internal_data_zip_md5}
+		)
+	")
+else()
+	install(DIRECTORY config DESTINATION ${DATA_DIR})
+	install(DIRECTORY Mods DESTINATION ${DATA_DIR})
+endif()
 if(ENABLE_LUA)
 	install(DIRECTORY scripts DESTINATION ${DATA_DIR})
 endif()
 
-# that script is useless for Windows and iOS
-if(NOT WIN32 AND NOT APPLE_IOS)
+# that script is useless for Windows / iOS / Android
+if(NOT WIN32 AND NOT APPLE_IOS AND NOT ANDROID)
```

Maybe check for Unix/Linux instead?
```suggestion
if(UNIX)
```

---

iOS and Android are definitely Unix :) it basically applies to "desktops that are not Windows", I'd keep as is.